tools/cache_creator/cache_info.cpp

```diff
@@ -225,7 +225,9 @@ llvm::raw_ostream &operator<<(llvm::raw_ostream &os, const BinaryCacheEntryInfo
   const vk::BinaryCacheEntry &header = *info.entryHeader;
 
   os << "\t*** Entry " << info.idx << " ***\n"
-     << "\thash ID:\t\t" << llvm::format_bytes(header.hashId.bytes, llvm::None, 16, 1) << "\n"
+     << "\thash ID:\t\t"
+     << llvm::format_hex(*((uint64_t*)header.hashId.bytes), 18) << ' '
+     << llvm::format_hex(*((uint64_t*)(header.hashId.bytes + 8)), 18) << '\n'
```

Comment:
You can access these two ints with `header.hashId.qwords[0]` and `header.hashId.qwords[1]`.

Reply:
uint64 has 8 bytes and each byte group is 2 characters wide, so I think it's enough to specify the width to be `sizeof(uint64_t) * 2` ==> `16`: https://github.com/llvm/llvm-project/blob/8652c3e1a3731052c74c82fe831773fc40b7b17e/llvm/include/llvm/Support/Format.h#L185

Reply:
Ah, thanks! I didn't realize it was a `MetroHash` type, that's much clearer. Hopefully the prefix change is too.